pdf/config.json5

```diff
@@ -35,6 +35,84 @@
         "example-file": {
             // Any disabled parser has no effect -- comment this line to enable
             disabled: true,
+            /** The command to invoke this parser.
+             *
```

Comment:
Just noticed I changed this comment format to have more asterisks. I have no preference and can change it back if anyone cares.

Reply:
Not enough to change it :)